index.yaml

```diff
@@ -391,3 +391,43 @@ indexes:
   properties:
   - name: feature_id
   - name: set_on
+- kind: Stage
+  properties:
+  - name: stage_type
+  - name: milestones.desktop_first
+  - name: feature_id
+- kind: Stage
+  properties:
+  - name: stage_type
+  - name: milestones.desktop_last
+  - name: feature_id
+- kind: Stage
+  properties:
+  - name: stage_type
+  - name: milestones.android_first
+  - name: feature_id
```

Comment:
Nice, I didn't know we could reference the subproperties like this 🙂

Reply:
I didn't either.  I ran it on staging and went with the suggested indexes that were logged on failures, then repeatedly pasted and edited it.